src/main/java/mekanism/common/lib/multiblock/StructureHelper.java

```diff
@@ -21,15 +21,16 @@ private StructureHelper() {
      * @param structure structure to check
      * @param minBounds minimum size of the cuboid
      * @param maxBounds maximum size of the cuboid
+     * @param filled if true, the sides must be filled, otherwise, just framed
      *
      * @return found cuboid, or null if it doesn't exist
      */
-    public static VoxelCuboid fetchCuboid(Structure structure, VoxelCuboid minBounds, VoxelCuboid maxBounds) {
+    public static VoxelCuboid fetchCuboid(Structure structure, VoxelCuboid minBounds, VoxelCuboid maxBounds, boolean filled) {
```

Comment:
This method should not be used for this purpose as the entire point is for if it is all sides, the other method is much better used for cases like this as it was designed for non rectangular multiblocks and multiblocks that are not "full" for each face such as the thermal evaporation tower.